hack/local-up-cluster.sh

```diff
@@ -27,6 +27,8 @@ ALLOW_SECURITY_CONTEXT=${ALLOW_SECURITY_CONTEXT:-""}
 PSP_ADMISSION=${PSP_ADMISSION:-""}
 NODE_ADMISSION=${NODE_ADMISSION:-""}
 RUNTIME_CONFIG=${RUNTIME_CONFIG:-""}
+# RUNTIME_CONFIG=${RUNTIME_CONFIG:-"settingsio/v1alpha1=true,settings.k8s.io/v1alpha1/podpreset=true"}
+RUNTIME_CONFIG=${RUNTIME_CONFIG:-"api/all=true"}
```

Comment:
This file got checked in by mistake. I have cleaned it up and enabled "api/all" by default.  PTAL.